### PR TITLE
Bump pysaml2

### PR DESCRIPTION
**Pull request by lgarvey**
Fixes several high and medium CVEs against pysaml2 and djangosaml2

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,14 +1,30 @@
-# SHA1:12a0ec44d6bb8638ef3ff4dffd549390f96e3eae
 #
-# This file is autogenerated by pip-compile-multi
+# This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile-multi
+#    pip-compile --output-file requirements-dev.txt requirements-dev.in
```
Comment (peterdemin) [inline requirements-dev.txt]: Why did you change the tool? I mean, were there any issues with `pip-compile-multi`?

Comment (lgarvey) [reply requirements-dev.txt]: I wasn't aware of `pip-compile-multi` will check it out.  Obviously the previous dev was using it.

Comment (peterdemin) [reply requirements-dev.txt]: Makes sense. Let me know if you face any questions/issues with pip-compile-multi.
